src/sst/core/watchPoint.cc

```diff
@@ -9,6 +9,7 @@
 // information, see the LICENSE file in the top level directory of the
 // distribution.
 
+//clang-format off
```

Comment:
Why is clang-format turned of here?  They seem to be in the order specified in the .clang-format file.